test/sql/chunk_utils.sql

```diff
@@ -318,12 +309,12 @@ BEGIN;
     \ir  :QUERY_RESULT_TEST_EQUAL_RELPATH
     \set ECHO all
     SELECT * FROM test.show_subtables('drop_chunk_test_ts');
-    SELECT show_chunks(hypertable => 'drop_chunk_test_tstz');
-    SELECT show_chunks(hypertable => 'drop_chunk_test_tstz', older_than => now() - interval '1 minute', newer_than => now() - interval '6 minute');
-    SELECT show_chunks(hypertable => 'drop_chunk_test_tstz', newer_than => now() - interval '1 minute');
-    SELECT show_chunks(hypertable => 'drop_chunk_test_tstz', older_than => now() - interval '1 minute');
+    SELECT show_chunks('drop_chunk_test_tstz');
+    SELECT show_chunks('drop_chunk_test_tstz', older_than => now() - interval '1 minute', newer_than => now() - interval '6 minute');
+    SELECT show_chunks('drop_chunk_test_tstz', newer_than => now() - interval '1 minute');
+    SELECT show_chunks('drop_chunk_test_tstz', older_than => now() - interval '1 minute');
 
-    \set QUERY1 'SELECT show_chunks(older_than => interval \'1 minute\', hypertable => \'drop_chunk_test_tstz\')::NAME'
+    \set QUERY1 'SELECT show_chunks(older_than => interval \'1 minute\', hypertable_or_cagg => \'drop_chunk_test_tstz\')::NAME'
```

Comment:
To match the `drop_chunks` call below.
```suggestion
    \set QUERY1 'SELECT show_chunks(\'drop_chunk_test_tstz\', interval \'1 minute\')::NAME'
```

Reply:
I think is is intentional to test different parameter order. In any case, this was there before.

Reply:
I made changes here before but didn't update calls to `show_chunks` since it used different parameters. IIRC, the original code looked the same for `show_chunks` and `drop_chunks` and I also believe there is a comment about that somewhere.

Reply:
Can you explain why the parameter order is important and why both calls need to look the same? The current order is still a valid way to call this function so I am not sure I see a problem with what's there now beyond maybe visual preference.

Reply:
AFAICT, it is not important, but it seems it was to the original author.